apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-picker-search-term-handler.js

```diff
@@ -4,14 +4,23 @@
  * External dependencies
  */
 import { debounce } from 'lodash';
+import debug from 'debug';
 
 /**
  * Internal dependencies
  */
 import tracksRecordEvent from './track-record-event';
 
+const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
+
+let previewTerm;
```

Comment:
I was little confused about the name of this variable, specially about the "preview" word. If this does what I think it does, would something like `lastSearchTerm` be better suited?

Reply:
oh 🤦‍♂️ , it's a typo. `previousTerm` was what I thought. What do you think?

Reply:
That's good too, sure! :)